opengemini_client/client_impl.py

```diff
@@ -158,7 +158,19 @@ def _query_post(self, query: Query) -> QueryResult:
         raise HTTPError(f"Query code: {resp.status_code}, body: {resp.text}")
 
     def write_batch_points(self, database: str, batch_points: BatchPoints):
-        return
+        server_url = self.get_server_url()
+        params = {'db': database}
+        with io.StringIO() as writer:
+            for bp in batch_points.points:
```

Comment:
calling every unit of batch_points as bp is bewildering ，could u replace it as point？